email-sending-concepts/domain-reputation.md

```diff
@@ -94,19 +94,20 @@ To build and maintain a strong domain reputation, consider these best practices:
 
 - **Authenticate your emails:** Set up [SPF](/email-sending-concepts/spf), [DKIM](/email-sending-concepts/dkim), and [DMARC](/email-sending-concepts/dmarc) to show mailbox providers your messages are trustworthy.
 
-- **Practice good list hygiene:** Regularly remove [hard bounces](/email-sending-concepts/hard-bounce) from your mailing list, and send emails only to recipients who have requested or expect them.
+- **Practice good list hygiene:** Regularly remove [hard bounces](/email-sending-concepts/hard-bounce) and inactive recipients from your mailing list. Immediately suppress addresses that hit the spam button.
 
 - **Focus on valuable content:** Create messages that your audience will find relevant and engaging.
 
+- **Encourage positive engagement:** Opens, clicks, and **replies** to emails have a direct positive impact on your domain reputation.
+
 - **Maintain a consistent sending pattern:** Try to keep your sending volume steady and avoid large, sudden increases.
 
 - **Monitor your performance:** Keep an eye on your key metrics so you can quickly identify and address any issues.
 
-- **Make unsubscribing easy:** Always include a clear, [one-click unsubscribe](/email-sending-concepts/one-click-unsubscribe) link to help minimize complaints.
+- **Make unsubscribing easy:** Always include a clear [one-click unsubscribe](/email-sending-concepts/one-click-unsubscribe) link. If recipients proceed to the unsubscribe page, present an option to "unsubscribe from all lists" so users can easily opt out from all communications if they wish.
```

Comment:
hmm... one-click unsubscribe link is provided by the inbox provider, if we add the necessary headers...